js/app/src/Blocks.svelte

```diff
@@ -420,6 +437,53 @@
 					if (!handled_dependencies[i]) handled_dependencies[i] = [];
 					handled_dependencies[i].push(id);
 				});
+
+			// share events
+			const FORM_INPUTS = [
+				"checkbox",
+				"dropdown",
+				"number",
+				"textbox",
+				"radio",
+				"slider"
```

Comment:
This is not super maintainable especially with community components down the road. This is basically why we were designing the mixin to get the serialized values from the components

Reply:
I'm wondering whether we should even support custom titles at this point? Perhaps for the v1, we simply use the filename as the custom title and then allow customization down the road depending on usage of the feature, WDYT?